workspaces/tests/gas_meter.rs

```diff
@@ -315,3 +228,61 @@ async fn test_dropped_gas_meter() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+async fn analogous_to_dev_deploy(
+    worker: &Worker<Sandbox>,
+    total_gas: &mut NearToken,
+) -> anyhow::Result<Contract> {
+    let account = worker
+        .root_account()?
+        .create_subaccount("contract")
+        .transact()
+        .await?;
+
+    *total_gas = total_gas
+        .checked_add(
+            account
+                .details
+                .outcomes()
+                .iter()
+                .fold(NearToken::from_yoctonear(0), |acc, receipt| {
+                    acc.checked_add(as_near(receipt.gas_burnt)).unwrap()
+                }),
+        )
+        .unwrap();
+
+    Ok(account
+        .into_result()?
+        .deploy(include_bytes!("../../examples/res/status_message.wasm"))
+        .await?
+        .into_result()?)
+}
+
+async fn analogous_to_dev_create_account(
+    worker: &Worker<Sandbox>,
+    total_gas: &mut NearToken,
+) -> anyhow::Result<Account> {
+    let account = worker
+        .root_account()?
+        .create_subaccount("alice")
+        .transact()
+        .await?;
+
+    *total_gas = total_gas
+        .checked_add(
+            account
+                .details
+                .outcomes()
+                .iter()
+                .fold(NearToken::from_yoctonear(0), |acc, receipt| {
+                    acc.checked_add(as_near(receipt.gas_burnt)).unwrap()
+                }),
+        )
+        .unwrap();
+
+    Ok(account.into_result()?)
+}
+
+const fn as_near(gas: Gas) -> NearToken {
+    NearToken::from_yoctonear(gas.as_gas() as u128)
+}
```

Comment:
What is going on here? You should not turn Gas into NearToken. Those are completely different things.